sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/bimanual_shl_ur10e.yaml

```diff
@@ -0,0 +1,42 @@
+robot:
```

Comment:
Should we name this file: bimanual_sh_left_ur10e.yaml  and the other bimanual_sh_right_ur10e.yaml? I think it looks cleaner